### ISSUE TITLE
How to add negative training data to prevent overfitting?

### ISSUE THREAD

**Issue by Edwardmark** [ultralytics/yolov5#1183]
## ❔Question

Hi, I want to know how to add negative training data (which contains no objects of interest) to train yolo-v5, how to arange the label /image file in such setting.
Thanks.

## Additional context
For example, I am training a model to detect bike, but lots of false positive(like motobike, wheels etc) is detected, So I collect many negative images, so how to add these to training as negative?

**Comment by TaoXieSZ** [comment]
You can use focal loss built in this repo

**Comment by glenn-jocher** [comment]
@Edwardmark you just add them to your dataset.

**Comment by Edwardmark** [comment]
@glenn-jocher so I do not add label to label folder, but just add the negative image to image folders? Is that true?

**Comment by glenn-jocher** [comment]
@Edwardmark sure, you just throw any images you want into your /images folder, they don't need labels.

**Comment by glenn-jocher** [comment]
@Edwardmark but of course, naturally you should verify that none of the unlabelled images you add contain items from your class list.

**Comment by Edwardmark** [comment]
@glenn-jocher Thanks, I get it.
